### ISSUE TITLE
Add eslint-plugin-lit-a11y to Tools/Development

### ISSUE THREAD

**Issue by justinfagnani** [lit/lit.dev#352]
https://www.npmjs.com/package/eslint-plugin-lit-a11y is an ESLint plugin that lints Lit template for a11y rules.

@thepassle is the README the best doc to point to?

**Comment by thepassle** [comment]
Yeah either the readme or this is fine: https://open-wc.org/docs/linting/eslint-plugin-lit-a11y/overview/

Thanks for adding it to the docs 🙂
